tools/evaluate_queries.cpp

```diff
@@ -199,7 +199,7 @@ int main(int argc, const char** argv)
 
     CLI11_PARSE(app, argc, argv);
 
-    tbb::task_scheduler_init init(app.threads());
+    tbb::global_control control(tbb::global_control::max_allowed_parallelism, app.threads());
```

Comment:
Looking at the docs, it seems like the number of threads include the main thread:

> Limit total number of worker threads that can be active in the task scheduler to parameter value - 1

So I'm wondering whether we should increase it to `app.threads() + 1` since our main thread only waits for the others, so effectively you'd get one fewer worker than you'd expect. Does this make sense?

Reply:
Yep agreed. I have also explicitly stated "worker threads" in the printout (which is the passed thread count, ignoring the +1).